models/Facility.yaml

```diff
@@ -0,0 +1,62 @@
+title: Facility
+x-stoplight:
+  id: mnxujfgg67otu
+type: object
+description: 'A facility is the enclosing building of a data center facility. This data model represents a single facility and can have many halls, cages, racks and servers.'
+properties:
+  id:
+    type: string
+    format: uuid
+    readOnly: true
+  name:
+    type: string
+    description: 'Optional: Give the facility a name'
+    example: Oregon 1
+  geo_latitude:
+    type: number
+    format: float
+    description: Precise latitude & longitude of the data center used to calculate emissions for the supplied electricity.
```

Comment:
How about cases where the facilities don't use energy from the national grid? Like most google DCs use renewable energy feed OR in other cases where a small-scale utility plant is used for power generation on-premise or shared between multiple parties in an industrial park?